libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -405,6 +405,43 @@ void AP_Vehicle::publish_osd_info()
 }
 #endif
 
+#ifdef ENABLE_SCRIPTING
```

Comment:
Don't you need to check if there is also GCS being included in the build? Remember periph has scripting these days but may not have MAVLink.